### PR TITLE
post: cli 7.2

### PR DESCRIPTION
**Pull request by cexbrayat**
No rush to review, the release is not there yet.

- [x] update date

### CODE REVIEW

File: _posts/2018-12-21-angular-cli-7.2.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+It can be a great help along the official `ng update @angular/core @angular/cli` command.
+You have no excuse for staying behind anymore!
+
+Let's see what we've got!
+
+## New flag and flag values for ng build
+
+### new resourcesOutputPath option
+
+It is now possible to specify where style resources will be placed, relative to `outputPath`.
```
[the remark this answers is not in the document]
Comment (cexbrayat) [reply _posts/2018-12-21-angular-cli-7.2.md]: it's not only CSS, it's style related resources, like fonts for example. I'll reword.

File: _posts/2018-12-21-angular-cli-7.2.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+- `evalSourceMap` in `ng build/serve` as it could be used to improve build performances, but isn't needed anymore.
+- `vendorSourceMap` in `ng build/serve`, as it has been replaced, see above.
+- `skipAppShell` in `ng build/serve` for a Web application as it had no effect.
+
+## Flag override warning
+
+Note that you will now have a warning when you specify a flag several times in the same command.
+It used to pick the last one silently,
+and now it will still do the same and also warn you that there is something weird going on:
```
[the remark this answers is not in the document]
Comment (cexbrayat) [reply _posts/2018-12-21-angular-cli-7.2.md]: Yeah, I meant it doesn't change the fact that it picks the last one, but it now warns you.

**Comment by cexbrayat** [comment]
I pushed a new commit with a few new flags

File: _posts/2018-12-21-angular-cli-7.2.md [hunk 3]
```diff
@@ -55,6 +55,20 @@ Now you can activate one or the other, as `optimization` can now take an object
       "styles": true
     }
 
+### Build Event Log
+
+A new flag called `buildEventLog` has been added to `ng build`.
+This flag needs a filename as a parameter and will generate the specified file,
+with a timeline of the build events.
+Currently, running `ng build --build-event-log bep.txt` generates:
```
Comment (clacote) [inline _posts/2018-12-21-angular-cli-7.2.md]: I was wondering at first why "bep" in `bep.txt", thinking it was a typo for "bel" as in Build Event Log. But reading later it probably means Build Event Protocol.

Comment (cexbrayat) [reply _posts/2018-12-21-angular-cli-7.2.md]: Right, I renamed to `log.txt` to be clearer

**Pull request merged**